packages/contracts/test/consent.ts

```diff
@@ -196,6 +196,50 @@ describe("Consent Contract and Factory Tests", function () {
       );
     });
 
+    // Change xit to it.only to test
+    xit("Testing computeFee limit", async function () {
+      const { consentFactory, token, owner, otherAccount } = await loadFixture(
+        deployConsentStack,
+      );
+
+      // ComputeFee's max returnable value is a uint256
+      // We can then try to figure out what is the max slot that it can take as an input
+      const maxUintTokens = ethers.MaxUint256;
+      const base = 1.0001;
+
+      // To calculate the number of tokens to stake, the ContentFactory's computeFee does 1.0001^(slot)
+      // So to determine the max slot given the maximum output of a uint256, we can use a reverse exponential calculation formula
+      // Using base of 1.0001 and maxUintTokens gives us a max slot value of 1774545.503594127
+      function reverseExponent(base, result) {
```

Comment:
Isn't a reverseExponent a logarithm?

Reply:
Mixed up my calculation namings! 

Dropping definition as future reference: "Logarithm is the operation that finds the exponent (our slot) given the base and the result, while reverse exponentiation (or finding the root) is the operation that finds the base given the exponent and the result."